lib/scrabble.rb

```diff
@@ -1,18 +1,51 @@
 class Scrabble
 
+  def initialize
+    @array_of_word = []
+    @count = 0
+  end
+
   def score(word)
-    1
+    if word.length > 6
+      @count += 10
+    end
+    @array_of_word =[]
+    word.split('').each do |letter|
+      @array_of_word << letter.upcase
+    end
+      point_counter(@array_of_word)
+
+  end
+
+  def point_counter(array)
+    points =  array
```

Comment:
Here you are setting a variable to another variable but you don't ever use the original array variable. I recommend next time putting points as your parameter/variable and referencing points from there.